.github/pull_request_template.md

```diff
@@ -4,6 +4,9 @@
 
 If you haven't already, read the full [contribution guide](../CONTRIBUTING.md). The guide may have changed since the last time you read it, so please double-check. Once you are done and ready to submit your PR, edit the PR description and run through the relevant checklist below.
 
+Enable GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).
+To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://stackoverflow.com/a/69063453/1341806) as a secret in your forked repository called `PAT`.
```

Comment:
I'd suggest using the official instructions for creating a PAT instead of a SO answer: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.